Drv/LinuxSpiDriver/LinuxSpiDriverComponentImpl.hpp

```diff
@@ -33,6 +33,30 @@ namespace Drv {
        SPI_FREQUENCY_20MHZ = 20000000UL,
     };
 
+    /**
+     * SPI Mode Select
+     *
+     * Defines the SPI Clock Polarity and Phase for each SPI Transaction.
+     * 
+     * SPI Clock Polarity(CPOL): Defines clock polarity as idle low (CPOL = 0) or idle high(CPOL = 1)
+     * SPI Clock Phase(CPHA): Defines if data is shifted out on the rising clock edge and sampled 
+     *                        on the falling clock edge(CPHA = 0) or if data is shifted out on the 
+     *                        falling clock edge and sampled on the rising clock edge(CPHA=1)
+     * 
+     * SPI Mode 0: (CPOL = 0, CPHA = 0) 
```

Comment:
I would take these SPI comments out and annotate the `SpiMode` enum with a description of what each means, since the "SPI Mode X" values are now hidden.